packages/cardano-services/src/ChainHistory/DbSyncChainHistory/types.ts

```diff
@@ -97,6 +98,8 @@ export interface ScriptModel {
   bytes: Buffer;
   hash: Buffer;
   serialised_size: number;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  json: any;
```

Comment:
`any` can be used as an `Object` while `unknown` can't be.
We know db-sync provides the data as a JSON Object.

Reply:
yes, that is the point for type safety. I am not sure where `json` is used, but ideally it would be `unknown` to force developers to type check before using it

Reply:
Being the source of this data db-sync, we know it returns an `Object`.
Adding a check which we know it will never fail sounds to me as a resources waste both in implementing and in running it.